doc/nrf/libraries/modem/nrf_modem_lib/nrf_modem_lib_lte_net_if.rst

```diff
@@ -0,0 +1,17 @@
+.. _nrf_modem_lib_lte_net_if:
+
+LTE Network Interface
```

Comment:
`LTE network interface` is the name of the Zephyr network interface driver for the nrf91 Series modem firmware.

I would call the chapter `Zephyr network interface driver` or `Network interface driver`, rather than `LTE network interface`, and begin explaining with:

```
The LTE network interface in the Modem library is a Zephyr network interface driver for the nRF91 Series cellular firmware.
By enabling the LTE network interface in the Modem library, you can control the nRF91 series modem as a Zephyr network interface, for example, :c:func:`net_if_up` and :c:func`net_if_down`.
```